lib/workos/sso.rb

```diff
@@ -67,12 +73,19 @@ def authorization_url(
         domain: nil,
         provider: nil,
         connection: nil,
+        organization: nil,
         state: ''
       )
+        if domain
+          warn '[DEPRECATION] `domain` is deprecated.
+          Please use `organization` instead.'
```

Comment:
Just thinking it'd be nice if had a pre-written block here. Some folks are going to hit this when updating their SDKs and not know how to proceed.

Reply:
Sorry, I'm not sure I follow. What do you mean by pre-written block? Just more information in the warning message?

Reply:
Sorry, sometimes my brain subs in the wrong word in the middle of sentences if I'm thinking about the next sentence while writing 😆 

Was thinking it'd be nice if we had a pre-written migration guide of this upgrade that we could link to from these warnings. All the best deprecation warnings I've ever had have a guide of what to do next. I might not know how to get the `organization` to pass in here.

I know we're not quite there yet in this migration, but thinking this is something we could backfill once written.

Reply:
I'm going to look into updating the docs today, so I could probably add a page that we can link to with more information.

Reply:
Awesome! I'll go ahead and merge this in, but should I hold off on a new version until we have a docs update and update the warning here?